src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$day-h-w: 100px;
+$day-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width:
+  $day-h-w * 7
+  + $day-gap * 6
+  + $calendar-padding * 2;
```

Comment:
I'd put it in 1 line :)
but this way is also good

Reply:
I just find it clearer this way. Thank you @mgrinko